### PR TITLE
adds ANI1x as dataset option

### PR DESCRIPTION
**Pull request by sebastianmdick**
I took the self_energies from the ANI1 datasets. Given that ANI1x uses a different XC functional, the atomic 'self energies' should in principle be adjusted to reflect that.

### CODE REVIEW

File: torchmdnet/datasets/ani1x.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+au_to_eV = 27.211386246
+
+
+class ANI1X(InMemoryDataset):
+
+    raw_url = "https://figshare.com/ndownloader/files/18112775"
+
+    element_numbers = {"H": 1, "C": 6, "N": 7, "O": 8}
+
+    self_energies = {
```
Comment (raimis) [inline torchmdnet/datasets/ani1x.py:19]: Could you add a comment the atomic energies are from the ANI-1 dataset.

File: torchmdnet/datasets/__init__.py [hunk 2]
```diff
@@ -1,8 +1,10 @@
+from torchmdnet.datasets.ani1x import ANI1X
```
Comment (PhilippThoelke) [inline torchmdnet/datasets/__init__.py:1]: Why import it twice?

**Review by PhilippThoelke — COMMENTED** [review]
Looks good but I'm confused by the double import in `__init__.py`.

File: torchmdnet/datasets/__init__.py (hunk 2 above)
Comment (sebastianmdick) [reply torchmdnet/datasets/__init__.py:1]: I think that was added by vscode. I must have missed it.

**Review by raimis — APPROVED** [review]

**Pull request merged**